packages/graphql/src/schema/resolvers/wrapper.ts

```diff
@@ -122,7 +122,8 @@ export const wrapResolver =
                         jwtDefault: new Cypher.NamedParam("jwtDefault", {}),
                     };
                 } catch (e) {
-                    if (authorization.globalAuthentication) {
+                    const globalAuthentication = schemaModel.annotations.authentication;
+                    if (authorization.globalAuthentication || globalAuthentication) {
                         throw e;
                     }
```

Comment:
This check needs to be removed from the wrapper entirely does it not? The check is now done in each translate function.